sdk/digitaltwins/azure-mgmt-digitaltwins/tests.yml

```diff
@@ -0,0 +1,15 @@
+trigger: none
```

Comment:
I don't think we run any live tests for mgmt plane libraries, and this file is not needed.

Reply:
Removed file